File: RentABikeWTR_v1_7.Model/Poslovnice.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace RentABikeWTR_v1_7.Model
{
    public class Poslovnice
    {
        public int PoslovnicaId { get; set; }
        public string NazivPoslovnice { get; set; }
        public string EmailKontakt { get; set; }
        public string Adresa { get; set; }
        public string BrojTelefona { get; set; }

        public ICollection<Bicikli> Bicikli { get; set; }
        public ICollection<PoziviDezurnomVozilu> PoziviDezurnomVozilu { get; set; }
    }
}

File: rentabikewtr_mobile/lib/model/poslovnice.dart
import 'dart:ffi';

import 'package:json_annotation/json_annotation.dart';

part 'poslovnice.g.dart';

@JsonSerializable()
class Poslovnice {
  int? poslovnicaId;
  String? nazivPoslovnice;
  String? emailKontakt;
  String? adresa;
  String? brojTelefona;

  Poslovnice() {}

  factory Poslovnice.fromJson(Map<String, dynamic> json) =>
      _$PoslovniceFromJson(json);

  /// Connect the generated [_$PoslovniceToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$PoslovniceToJson(this);
}
